### ISSUE TITLE
Tempature not shown

### ISSUE THREAD

**Issue by Bram-diederik** [tobias-kuendig/hacompanion#16]
I got hacompanion running on my raspberry pi vm server (mouse) and my primary laptop. (Blue-mage)

But tempature is not shown. Perhaps its configuration but i dont see any configuration mentions in the readme.
![Screenshot_20220811-123201_JuiceSSH](https://user-images.githubusercontent.com/53519837/184115863-ba72bfc1-7092-4e14-af5b-3d4a9b6e1e2d.jpg)
![Screenshot_20220811-123118_JuiceSSH](https://user-images.githubusercontent.com/53519837/184115870-40025ba6-1898-41fc-95b8-7b5e204821a0.jpg)
![Screenshot_20220811-121833_Home Assistant](https://user-images.githubusercontent.com/53519837/184115875-303a78de-b6eb-4dd2-89ae-529b4f142fd0.jpg)

Btw you closed #15 the quote was there because i 1st tryed to run it with out the listener. I think it runs port 80 it there is no configuration so you cant turn off the listener.

**Comment by tobias-kuendig** [comment]
The Regex to parse the cpu temp currently only looks fore `core` or `Package id` but not for `CPU`, this could be easily fixed:

https://github.com/tobias-kuendig/hacompanion/blob/d7b3431aa96bef742700311d0a44c64545c12318/sensor/cpu.go#L19

**Comment by jackyaz** [comment]
I fixed this on my Pi with this change

```
--- a/sensor/cpu.go
+++ b/sensor/cpu.go
@@ -16,7 +16,7 @@ import (
 )
 
 var (
-       reCPUTemp  = regexp.MustCompile(`(?m)(Package id|Core \d)[\s\d]*:\s+.?([\d\.]+)°`)
+       reCPUTemp  = regexp.MustCompile(`(?m)(temp1|Package id|Core \d)[\s\d]*:\s+.?([\d\.]+)°`)
        reCPUUsage = regexp.MustCompile(`(?m)^\s*cpu(\d+)?.*`)
 )
 
@@ -54,7 +54,7 @@ func (c CPUTemp) process(output string) (*entity.Payload, error) {
                if len(match) < 3 {
                        return nil, fmt.Errorf("invalid output form lm-sensors received: %s", output)
                }
-               if strings.EqualFold(match[1], "Package id") {
+               if strings.EqualFold(match[1], "Package id") || strings.EqualFold(match[1], "temp1") {
                        p.State = match[2]
                } else {
                        p.Attributes[util.ToSnakeCase(match[1])] = match[2]
```

**Comment by Bram-diederik** [comment]
how do i build a go project? i tried go build but that did not work.

**Comment by jackyaz** [comment]
You can try the steps here: https://github.com/jackyaz/hacompanion#installation

Depending on your platform, you may need to install a newer version of Go than provided by your package manager. https://go.dev/doc/install

**Comment by Bram-diederik** [comment]
strange i compiled it for my amd64 it works. but when i crosscompiled it for my PI it did not work. program runs cpu not shown.

**Comment by jackyaz** [comment]
Was that compiling my fork or this one?
I didn't cross-compile, I built it on the Pi directly

**Comment by Bram-diederik** [comment]
Did not run on pi correctly due to configuration all is fine now.

**Comment by pschmitt** [comment]
Just opened a PR that will probably fix that: #38

**Comment by tobias-kuendig** [comment]
fixed in #38
